docker-compose-web-modeler-beta.yaml

```diff
@@ -0,0 +1,165 @@
+# [2] - https://camunda.com/legal/terms/camunda-platform/camunda-platform-8-self-managed/
+# [3] - https://forum.camunda.io/
+
+# Notes: this file is using mailhog to simulate a Mail server
+
+# TODO 
+# - swap latest tag with release version
+
+version: "2.4"
+
+services:
+
+  modeler-db:
+    container_name: modeler-db
+    image: postgres:14.2-alpine
+    ports:
+      - "5433:5432"
+    healthcheck:
+      test: pg_isready -d modeler-db -U modeler-db-user
+      interval: 5s
+      timeout: 15s
+      retries: 30
+    environment:
+      POSTGRES_DB: modeler-db
+      POSTGRES_USER: modeler-db-user
+      POSTGRES_PASSWORD: modeler-db-password
+    networks:
+      - modeler      
+
+  modeler-websockets:
+    container_name: websockets
+    image: registry.camunda.cloud/cawemo-saas/cawemo-websockets:latest
+    ports:
+      - "8060:8060"
+    healthcheck:
+      test: /usr/bin/nc -v localhost 8060
+      interval: 30s
+    environment:
+      APP_NAME: "Web Modeler Self-Managed WebSockets"
+      APP_DEBUG: "true"
+      PUSHER_APP_ID: modeler-app
+      PUSHER_APP_KEY: modeler-app-key 
+      PUSHER_APP_SECRET: modeler-app-secret
+      PUSHER_APP_CLUSTER: modeler
+    networks:
+      - modeler      
+
+  mailhog:
+    # If you want to use your own SMTP server, you can remove this container
+    # and configure RESTAPI_MAIL_HOST, RESTAPI_MAIL_PORT and RESTAPI_MAIL_ENABLE_TLS
+    # accordingly
+    container_name: mailhog
+    image: mailhog/mailhog:v1.0.1
+    ports:
+      - "1025:1025"
+      - "8075:8025"
+    healthcheck:
+      test: /usr/bin/nc -v localhost 1025
+      interval: 30s  
+    networks:
+      - modeler                
+
+  # Modeler containers
+  modeler-restapi:
+    container_name: modeler-restapi
+    image: registry.camunda.cloud/web-modeler/modeler-restapi-self-managed:latest
+    ports:
+      - "8085:8081"
+      - "8086:8091"
+    command: /bin/sh -c "java $JAVA_OPTIONS org.springframework.boot.loader.JarLauncher"
+    depends_on:
+      modeler-db:
+        condition: service_healthy
+      mailhog:
+        condition: service_started  
+      identity:
+        condition: service_healthy      
+
+    healthcheck:
+      test: /usr/bin/nc -v localhost 8081
+      interval: 30s
+      retries: 10
+    environment:
+      JAVA_OPTIONS: -Xmx128m
+      LOGGING_LEVEL_IO_CAMUNDA_MODELER: DEBUG
+      SPRING_PROFILES_INCLUDE: default-logging  
+      RESTAPI_PUSHER_HOST: modeler-websockets
+      RESTAPI_PUSHER_PORT: "8060"
+      RESTAPI_PUSHER_APP_ID: modeler-app
+      RESTAPI_PUSHER_KEY: modeler-app-key
+      RESTAPI_PUSHER_SECRET: modeler-app-secret
+      RESTAPI_OAUTH2_TOKEN_AUDIENCE: web-modeler-api
+      RESTAPI_OAUTH2_TOKEN_ISSUER: http://localhost:18080/auth/realms/camunda-platform
+      RESTAPI_OAUTH2_TOKEN_ISSUER_BACKEND_URL: http://keycloak:8080/auth/realms/camunda-platform
+      RESTAPI_IDENTITY_BASE_URL: http://identity:8084/          
+      RESTAPI_SERVER_URL: http://localhost:8081
```

Comment:
`RESTAPI_SERVER_URL` has to use the exposed port of the webapp: `http://localhost:8070`
It is used, for example, to construct the URL for project invitations (which are opened in the browser).